hicommonwealth/commonwealth
# Fix poll duration

## Link to Issue
Closes: #5556

## Description of Changes
- Fixes poll duration

## "How We Fixed It"
- Simplify and refactor logic

## Test Plan
- Create a poll with default duration (not custom)– confirm that poll is 5 days
- Create poll with custom duration of 3 days– confirm that poll is 3 days
- Create poll with custom duration of Infinite– confirm that poll has no duration

## Deployment Plan
N/A

## Other Considerations
N/A
